drivers/filesystems/cdfs/CMakeLists.txt

```diff
@@ -38,3 +38,9 @@ target_link_libraries(cdfs ${PSEH_LIB} memcmp)
 add_importlibs(cdfs ntoskrnl hal)
 add_cd_file(TARGET cdfs DESTINATION reactos/system32/drivers NO_CAB FOR all)
 add_registry_inf(cdfs_reg.inf)
+
+if (CLANG AND GCC)
+    # Make __inline work as expected
+    target_compile_options(cdfs PRIVATE -std=gnu89)
```

Comment:
I wonder if we should change this default globally or something...

Reply:
No idea, really...